lib/utils/logger.js

```diff
@@ -24,6 +25,13 @@ class Logger {
     console.warn(`(${this.config.context}) Warning: ${message}`);
   }
 
+  inspect(value) {
+    if (this.config.inspector) {
```

Comment:
There is no need for this config, as there is currently no way sequelize let users pass this config. So they can't override this.

Proper way to override will be to just replace `logger.inspect` method that is exported by this module.